ocean_lib/web3_internal/web3_overrides/utils.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import time
+from typing import Optional
+
+from enforce_typing import enforce_types
+from hexbytes import HexBytes
+from web3 import Web3
+
+from ocean_lib.config import Config
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+@enforce_types
+def send_dummy_transactions(block_number: int, from_wallet: Wallet) -> None:
```

Comment:
Please add a docstring. I think it should include a detailed description of what the `block_number` parameter is supposed to be.

Reply:
Marking resolved because function no longer exists.